packages/forklift-console-plugin/src/modules/Providers/views/details/components/CredentialsSection/RevealedData.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { useForkliftTranslation } from 'src/utils/i18n';
+
+import { ClipboardCopy, ClipboardCopyVariant, TextInput } from '@patternfly/react-core';
+
+export const RevealedData: React.FC<{ value: string }> = ({ value }) => {
```

Comment:
hmm,  `RevealedData` explain when we use this component, but it does not explain what this component is, can you think of names that tell the user what the compoenent do ( e.g. table, input, ... ) ?

p.s.
add some jdoc?

Reply:
p.s.
now that it is a component we can add a prop to hint how handle different types of fields like, boolean, or long texts,  WDYT?